spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -412,6 +412,15 @@ def get_beta_tester_metrics(filter: {}, includes: nil, limit: nil, sort: nil)
           test_flight_request_client.get("betaTesterMetrics", params)
         end
 
+        #
+        # buildBundles
+        #
+
+        def get_build_bundles_build_bundle_file_sizes(build_bundle_id:, limit: nil)
```

Comment:
I went with `build_bundle_id` over `bundle_id` to avoid confusion with the bundle identifier (i.e com.my.app).